src/main/api/catalog.ts

```diff
@@ -71,64 +71,63 @@ export class CatalogApi implements ICatalogApi {
         const {
             audio: {
                 readed: audiobookReaded,
-                added: audiobookAdded,
             },
             divina: {
                 readed: divinaReaded,
-                added: divinaAdded,
             },
             epub: {
                 readed: epubReaded,
-                added: epubAdded,
             },
             pdf: {
                 readed: pdfReaded,
-                added: pdfAdded,
+            },
+            all: {
+                added: allAdded,
             },
         } = await this.getPublicationView();
 
         // Dynamic entries
         const entries: CatalogEntryView[] = [
-            {
-                title: __("catalog.entry.continueReading"),
-                totalCount: epubReaded.length,
-                publicationViews: epubReaded,
-            },
             {
                 title: __("catalog.entry.lastAdditions"),
-                totalCount: epubAdded.length,
-                publicationViews: epubAdded,
+                totalCount: allAdded.slice(0, NB_PUB).length,
```

Comment:
Shouldn't this be `Math.max(NB_PUB, allAdded.length)`?
(same remark with all the other cases in this PR)

Reply:
Sorry, brain fart :(